src/App.tsx

```diff
@@ -1,39 +1,60 @@
 import React from 'react';
 import './App.scss';
```

Comment:
It is a good practice to separate our components from other libraries by using 1 empty line